packages/platform-sdk/test/integrationTests/inventory/inventory.test.ts

```diff
@@ -0,0 +1,108 @@
+import { randomUUID } from 'crypto'
+import { apiRoot } from '../test-utils'
+import { ChannelResourceIdentifier, InventoryEntryDraft } from '../../../src'
+import { createChannel, deleteChannel } from '../channel/channel-fixture'
+import {
+  createInventory,
+  createInventoryDraft,
+  deleteInventory,
+} from './inventory-fixture'
+
+describe('testing inventory API calls', () => {
+  it('should create and delete an inventory by ID', async () => {
+    const channel = await createChannel()
+    const channelResourceIdentifier: ChannelResourceIdentifier = {
+      typeId: 'channel',
+      id: channel.body.id,
+    }
+
+    const inventoryDraft: InventoryEntryDraft = {
+      sku: randomUUID(),
+      supplyChannel: channelResourceIdentifier,
+      quantityOnStock: 10,
+    }
+
+    const responseCreatedInventory = await apiRoot
+      .inventory()
+      .post({ body: inventoryDraft })
+      .execute()
+
+    expect(responseCreatedInventory.statusCode).toEqual(201)
+    expect(responseCreatedInventory.body).not.toBe(null)
+
+    const responseInventoryDeleted = await apiRoot
+      .inventory()
+      .withId({ ID: responseCreatedInventory.body.id })
+      .delete({
+        queryArgs: { version: responseCreatedInventory.body.version },
+      })
+      .execute()
+
+    expect(responseInventoryDeleted.statusCode).toEqual(200)
+
+    await deleteChannel(channel)
+  })
+
+  it('should get an inventory by ID', async () => {
+    const channel = await createChannel()
```

Comment:
here is the same like previous comment: "Leaving these methods here to show the customer step-by-step how to create/delete the final object"